src/Symfony/Bundle/TwigBundle/TwigEngine.php

```diff
@@ -51,7 +51,8 @@ public function setDefaultEscapingStrategy($strategy)
     }
 
     /**
-     * @deprecated Deprecated since version 2.3, to be removed in 3.0. Use TwigDefaultEscapingStrategy instead.
+     * @deprecated since version 2.3, to be removed in 3.0.
```

Comment:
if it was deprecated in a patch version it was a mistake imo. we should not do that.